pkg/didcomm/packer/anoncrypt/pack.go

```diff
@@ -63,20 +61,27 @@ func New(ctx packer.Provider, encAlg jose.EncAlg) (*Packer, error) {
 	}, nil
 }
 
-// Pack will encode the payload argument
-// Using the protocol defined by the Anoncrypt message of Aries RFC 0334
-// Anoncrypt ignores the sender argument, it's added to meet the Packer interface.
-func (p *Packer) Pack(payload, _ []byte, recipientsPubKeys [][]byte) ([]byte, error) {
+// Pack will encode the payload argument using the protocol defined by the Anoncrypt message of Aries RFC 0334.
+// Anoncrypt ignores the sender argument, it's added to meet the Packer interface. It uses DIDComm typ and cty V2 values
+// (default envelope type and content type headers).
+func (p *Packer) Pack(contentType string, payload, _ []byte, recipientsPubKeys [][]byte) ([]byte, error) {
 	if len(recipientsPubKeys) == 0 {
 		return nil, fmt.Errorf("anoncrypt Pack: empty recipientsPubKeys")
 	}
 
+	switch contentType {
```

Comment:
I don't think contentType is the responsibility of the packer except for passing it through.

Reply:
ok, I'll remove this switch statement

Reply:
switch statement removed, now city is pass through in the packer and JWEEncrypter, if it is set, it will be added to the JWE protect headers.